pandas/tests/series/test_api.py

```diff
@@ -602,82 +602,6 @@ def f():
                                       ordered=True))
         tm.assert_series_equal(result, expected)
 
-    def test_str_accessor_api_for_categorical(self):
-        # https://github.com/pandas-dev/pandas/issues/10661
-        from pandas.core.strings import StringMethods
-        s = Series(list('aabb'))
-        s = s + " " + s
-        c = s.astype('category')
-        assert isinstance(c.str, StringMethods)
-
-        # str functions, which need special arguments
-        special_func_defs = [
```

Comment:
The full list of these functions & arg-combinations is reflected in the `any_string_method`-fixture of `test_strings.py` now.